src/app/core/entity-components/entity-subrecord/entity-subrecord/entity-subrecord.component.scss

```diff
@@ -3,26 +3,9 @@
 @use "@angular/material" as mat;
 
 .table-action-button {
-  font-size: 115%;
-  color: mat.get-color-from-palette(theme.$accent);
```

Comment:
We should stick to the color theme and display these actions in "accent" (i.e. blue). Now the button feels disabled even when it's not ;-)